app/styles/vendor/_flag-icon.scss

```diff
@@ -0,0 +1,5 @@
+$flag-icon-css-path: "~flag-icon-css/flags" !default;
```

Comment:
Do we need ~ for this import? I think the post-css-loader would make it unnecessary.

Reply:
Removal of ~ breaks the webpack build for css.